lib/manageiq/cross_repo/runner.rb

```diff
@@ -35,14 +37,32 @@ def run
 
     def run_tests
       with_test_env do
-        system!({"TRAVIS_BUILD_DIR" => test_repo.path.to_s}, "bash", "tools/ci/before_install.sh") if ENV["CI"] && File.exist?("tools/ci/before_install.sh")
-        system!(env_vars, "bin/setup")
-        system!(script_cmd)
+        commands =
```

Comment:
I can clean this up and automatically add travis folding comments if I loop through section names just need to ||= the defaults first for e.g. script_cmd